Dockerfile

```diff
@@ -19,6 +19,20 @@ RUN yum install -y nodejs npm java-11
 # Copy the go source
 COPY ["Makefile", "main.go", ".openapi-generator-ignore", "openapitools.json", "./"]
 
+
+# Download protoc compiler v24.3
+RUN set -ex\
+    ; ARCH=$(uname -m) ; echo $ARCH \
+    ; if [ "$ARCH" == "s390x" ] ; then ARCH="s390_64" ; elif [ "$ARCH" == "ppc64le" ] ; then ARCH="ppcle_64" ; fi \
+    ; wget -q https://github.com/protocolbuffers/protobuf/releases/download/v24.3/protoc-24.3-linux-$ARCH.zip -O protoc.zip \
+    ; unzip -q protoc.zip \
+    ; bin/protoc --version \
+    ; rm protoc.zip \
+    ;
```

Comment:
The download of protoc is already performed as part of `make deps` which includes `make bin/protoc`, in line below 34.

This was improved with #52 

Kindly consider extending `scripts/install_protoc.sh` for your platforms, so we don't need this extra Docker RUN ?

Reply:
i didn't know about the protoc change . Once it is approved , i will update my changes accordingly for protoc